src/contribute/index.md

```diff
@@ -215,6 +301,12 @@ Easy to use Java 8 DSL for the Beam Java SDK. Provides a high-level abstraction
 - JIRA: [dsl-euphoria](https://issues.apache.org/jira/browse/BEAM-4366?jql=project%20%3D%20BEAM%20AND%20component%20%3D%20dsl-euphoria) / [BEAM-3900](https://issues.apache.org/jira/browse/BEAM-3900) 
 - Contact: [David Moravek](mailto:david.moravek@gmail.com)
 
+### Improving the contributor experience
+
+Making it easier to write code, run tests, and release. Investigating using docker for jenkins builds, automating the release process, and improving the reliability of tests.
+
+Ideas and help welcome! Contact: [Alan Myrvold](mailto:amyrvold@google.com), [Mark Liu](mailto:markliu@google.com), [Yifan Zou](mailto:yifanzou@google.com)
```

Comment:
Not an objection, but a question.

Is there a more generic contact we can use instead of direct personal contact? The reason is we would not like to hot-spot these people forever, and we would like to exclude others.

Reply:
All of the other sections for Works in Progress list personal contacts.

We could create separate mailing lists like other projects do; several apache projects have more granular mailing lists, http://mail-archives.apache.org/mod_mbox/ but I don't think we have the volume to justify splitting off the dev list into works in progress lists.